src/content/blog/biome-v1-9.md

```diff
@@ -0,0 +1,242 @@
+```
+
+Special thanks to [Denis Bezrukov @denbezrukov](https://github.com/denbezrukov) and [Jon Egeland @faultyserver](https://github.com/faultyserver) for implementing most of the CSS parser and formatter!
+Special thanks to [Yoshiaki Togami @togami2864](https://github.com/togami2864) for coordinating the implementation and for implementing most of the CSS lint rules!
+
+### Stable GraphQL formatter and linter
+
+Another brand new feature: Biome now formats and lints GraphQL files by default.
+
+For now, Biome provides two nursery rules:
+
+- [nursery/noDuplicateFields](https://biomejs.dev/linter/rules/no-duplicate-fields/)
+- [nursery/useDeprecatedReason](https://biomejs.dev/linter/rules/use-deprecated-reason/)
+
+If you don't want Biome to format and lint your GraphQL files,
+you can disable the GraphQL formatter and linter in the Biome configuration file:
+
+```json
+{
+  "graphql": {
+    "formatter": {
+      "enabled": false
+    },
+    "linter": {
+      "enabled": false
+    }
+  }
+}
+```
+
+or on the command line:
+
+```shell
+biome format --graphql-formatter-enabled=false
+biome lint --graphql-linter-enabled=false
+biome check --graphql-formatter-enabled=false --css-linter-enabled=false
+```
+
+Special thanks to [Võ Hoàng Long @vohoanglong0107](https://github.com/vohoanglong0107) for implementing most of the features related to GraphQL.
+
+### Search command
+
+Back in February, one of our Core Contributors published [a proposal for plugin support](https://github.com/biomejs/biome/discussions/1762). One of the highlights was the use of GritQL as a foundation for our plugin system.
+
+[GritQL](https://docs.grit.io/language/overview) is a powerful query language that lets you do structural searches on your codebase. This means that trivia such as whitespace or even the type of string quotes used will be ignored in your search query. It also has many features for querying the structure of your code, making it much more elegant for searching code than regular expressions.
+
+Integrating a query language such as GritQL is no easy feat, and throughout the year we published [multiple](https://github.com/biomejs/biome/discussions/2286) [status](https://github.com/biomejs/biome/discussions/2585) [updates](https://github.com/biomejs/biome/discussions/3392). Today, we release the first product of this effort: A new `biome search` command.
+
+While we believe this command may already be useful to users in some situations (especially when it gets integrated in our IDE extensions!), this command is really a stepping stone towards our plugin efforts. By allowing our users to try it out in a first iteration, we hope to gain insight into the type of queries you want to do, as well as the bugs we need to focus on.
+
+For now, the search command is explicitly marked as **EXPERIMENTAL**, since many limitations are yet to be fixed or explored. Keep this in mind when you try it out, and please let us know what you think!
```

Comment:
I'm thinking it might be good to add one line still:

```suggestion
For an overview of specific limitations, please see: https://github.com/biomejs/biome/issues/2582
```